docs/core/deploying/trimming/trim-warnings/il2116.md

```diff
@@ -1,13 +1,15 @@
 ---
-title: "IL2116: 'RequiresUnreferencedCodeAttribute' cannot be placed directly on a static constructor. Consider placing it on the type declaration instead."
-description: "Learn about trim warning IL2116: RequiresUnreferencedCodeOnStaticConstructor"
+title: "Trim Warning IL2116: The use of 'RequiresUnreferencedCodeAttribute' on static constructors is disallowed since is a method not callable by the user, is only called by the runtime. Placing the attribute directly on the static constructor will have no effect, instead use 'RequiresUnreferencedCodeAttribute' on the type which will handle warning and silencing from the static constructor."
+description: "Learn about trim warning IL2116"
 ms.date: 10/06/2021
 ms.topic: reference
 author: mateoatr
 f1_keywords:
   - "IL2116"
 ---
-# IL2116: 'RequiresUnreferencedCodeAttribute' cannot be placed directly on a static constructor. Consider placing it on the type declaration instead
+# Trim Warning IL2116: The use of 'RequiresUnreferencedCodeAttribute' on static constructors is disallowed since is a method not callable by the user, is only called by the runtime. Placing the attribute directly on the static constructor will have no effect, instead use 'RequiresUnreferencedCodeAttribute' on the type which will handle warning and silencing from the static constructor.
+
+'RequiresUnreferencedCodeAttribute' cannot be placed directly on static constructor 'constructor', consider placing 'RequiresUnreferencedCodeAttribute' on the type declaration instead.
```

Comment:
```suggestion
`RequiresUnreferencedCodeAttribute` cannot be placed directly on static constructor 'constructor'. Consider placing `RequiresUnreferencedCodeAttribute` on the type declaration instead.

The use of `RequiresUnreferencedCodeAttribute` on static constructors is disallowed since static constructors are not callable by the user, only by the runtime. Placing the attribute directly on the static constructor has no effect. Instead, place `RequiresUnreferencedCodeAttribute` on the type, which will silence the warning from the static constructor.
```